docs/platform/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md

```diff
@@ -4,61 +4,66 @@ products: all
+| --------------------------------------- | -------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Failed syncs**                        | &#10003; | &#10003;     | A sync from any of your connections fails. If syncs runs frequently or if there are many syncs in the workspace, this event can be noisy.              |
+| **Successful syncs**                    | &#10003; | &#10003;     | A sync from any of your connections succeeds. If syncs runs frequently or if there are many syncs in the workspace, this event can be noisy.           |
+| **Connection updates**                  | &#10003; | &#10003;     | A connection is updated automatically. For example, a source schema is automatically updated.                                                          |
+| **Connection updates requiring action** | &#10003; | &#10003;     | A connection update requires you to take action. For example, a breaking change.                                                                       |
+| **Warning - repeated failures**         | &#10003; | &#10003;     | Airbyte is at risk of turning off a connection due to repeated failures. It has failed 20 times consecutively and his not been successful in the last 4 days. |
+| **Sync disabled - repeated failures**   | &#10003; | &#10003;     | Airbyte has turned off a connection due to repeated failures. It has failed 30 times consecutively and has not been successful in the last 7 days.     |
+| **Warning - upgrade required**          | &#10003; |              | A new connector version is available, but you need to manually upgrade.                                                                                |
+| **Sync Disabled - upgrade required**    | &#10003; |              | Airbyte turned off one or more connections automatically because you missed the deadline to upgrade the connector.                                     |
 
-## Configure Email Notification Settings
+### Enabling schema update notifications
 
-<AppliesTo cloud />
+If you want Airbyte to notify you of source schema changes, enable the following notifications.
 
-To set up email notifications, click **Settings** and navigate to **Workspace** > **Notifications**.
+- `Automatic Connection Updates`
+- `Connection Updates Requiring Action`.
 
-Toggle which messages you'd like to receive from Airbyte. All email notifications will be sent by default to the creator of the workspace.
+If these are off, even if you turned on schema update notifications in a connection's settings, Airbyte won't send notifications about schema changes.
 
-![](./assets/notifications-email.png)
+To edit this setting, click **Connections** and select the connection you want to receive notifications for. Click the **Settings** tab on the Connection page. In the **Advanced Settings**, toggle **Schema update notifications**.
```

Comment:
This is named slightly different in the connection advanced settings:

<img width="582" height="75" alt="image" src="https://github.com/user-attachments/assets/1eb75a11-1b5b-46ca-844a-b66de3eeb400" />